helix-term/src/keymap/default.rs

```diff
@@ -274,6 +274,11 @@ pub fn default() -> HashMap<Mode, KeyTrie> {
             "k" => hover,
             "r" => rename_symbol,
             "h" => select_references_to_symbol_under_cursor,
+            "c" => { "Toggle comments"
+                "c" => toggle_comments,
+                "l" => toggle_line_comments,
+                "b" => toggle_block_comments,
+            },
```

Comment:
How about using `space c` for toggle, `space C` for block, then `space alt-C` for block? That way we make it a simple press for the default usecase, and the other ones can be accessed with modifiers

Reply:
that seems good could you clarify which one you would like to be line comment?

Reply:
I changed it to being alt-c for line comment, no worries if you want it the other way but I imagine more people would use block comment binding then the line comment one.